### ISSUE TITLE
transitions through different rooms

### ISSUE THREAD

**Issue by CL0S3T-D3M0NZ** [AurySystem/bitsy-color#8]
i kinda just made an account on here solely for this lol, but anyway --
i noticed that when you transition through different rooms and it has a transiton effect/animation, the room kind of just goes blank for a few seconds and it looks glitchy. for the fading effects, the animation plays out but the room goes empty and _then_ transitions to the next room. same thing with the sliding effects/animations as well AND the wavy effect (except that one just make the room go blank for a few seconds, then transitions like the others). idk if its a bug or something but it just does that.
my wording is TERRIBLE but if you didn't understand any of it then a game i'm working on is an example of this lol: https://closet-demonz.itch.io/bittersweet-love

**Comment by AurySystem** [comment]
yeah that's weird, we need to edit transitions becuase of other funky stuff from upstream
